src/views/ProtectionHome.vue

```diff
@@ -274,6 +354,38 @@ export default class ProtectionHome extends BaseComponent {
     return `${start} - ${end}`;
   }
 
+  get anyAreFiltering() {
+    let anyFiltering: boolean = false;
+    this.dropDownFilters.map(
+      dropDown => (anyFiltering = dropDown.selectedIndex !== 0)
+    );
```

Comment:
What are we doing here? 
Seems like we're iterating over ```this.dropDownFilters``` and updating ```anyFiltering``` several times based on ```dropDown.selectedIndex !== 0``` which would just make the last element in the ```this.dropDownFilters``` array be the final decider wouldn't it? 

Also if we're not looking to build a new array we should favour ```forEach``` over ```map``` , as it's more clear. 

Is this what we're looking for?

```suggestion
    const anyFiltering = this.dropDownFilters.some(dropDown => dropDown.selectedIndex !== 0);
```

Reply:
This is outdated I redone the logic to it since we needed very different things for every filter,
The filters are separated now.
I do agree we need to favor foreach in those cases.